adapters/tappx/tappx.go

```diff
@@ -51,14 +51,58 @@ func (a *TappxAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapt
 			Message: "Error parsing bidderExt object",
 		}}
 	}
-
 	var tappxExt openrtb_ext.ExtImpTappx
 	if err := json.Unmarshal(bidderExt.Bidder, &tappxExt); err != nil {
 		return nil, []error{&errortypes.BadInput{
 			Message: "Error parsing tappxExt parameters",
 		}}
 	}
 
+	//In this section we create all EXT parameters
+	type VideoExt struct {
+		Rewarded int `json:"rewarded,omitempty"`
+	}
```

Comment:
This structure is unused.

Reply:
Hi! We resolve this in other comment. We now find useless to use the rewarded tappx param. 

We clean all this in a2ddc1f commit.